src/containers/offer-page/create-or-edit-offer/CreateOrEditOffer.constants.ts

```diff
@@ -10,7 +10,7 @@ export const getInitialValues = (offer: Offer | null) => ({
   description: offer?.description ?? '',
   price: offer?.price.toString() ?? '',
   status: offer?.status ?? StatusEnum.Active,
-  FAQ: offer?.FAQ ?? [{ question: '', answer: '' }]
+  FAQ: offer?.FAQ ?? [{ question: '', answer: '', _id: `${Date.now()}` }]
```

Comment:
Look. `el` item already have `_id` property, so we can we may not create another `_id` here

<img width="1279" alt="image" src="https://github.com/ita-social-projects/SpaceToStudy-Client/assets/56305508/1683cad3-461b-4e2c-a834-6ddd146082c7">